unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/ParcelScenesCleaner.cs

```diff
@@ -41,15 +41,9 @@ public MarkedSharedComponentInfo(ParcelScene scene, string componentId)
 
         Coroutine removeEntitiesCoroutine;
 
-        public ParcelScenesCleaner ()
+        public void Initialize ()
         {
             removeEntitiesCoroutine = CoroutineStarter.Start(CleanupEntitiesCoroutine());
-            CommonScriptableObjects.rendererState.OnChange += OnRendererStateChange;
```

Comment:
shouln't we keep this event subscription?
If we remove it we should remove its unsubcription from `Dispose()` and remove the `OnRendererStateChange()` implementation altogether.